### ISSUE TITLE
how to extract the image for train?first frame , random frame or all frames?

### ISSUE THREAD

**Issue by H-Kcloud** [ondyari/FaceForensics#46]
Hello ,
I am so confused，
I have some simple questions.  does the  training set extract all frames of the video, or does it choose the first frame or randomly extract a frame?
Then  after the training is completed, how to calculate the test results in the article ?
To test the video of the entire test set, and then count the majority of the classification results of each frame of this video,  get the majority  as the prediction of the video?Or  extract the first frame , random frame,or extract all frames to calculate all classification results? 
Thank you!

**Comment by liquor233** [comment]
I think: when train, you can extract all or random frames as you like, when test, extract all frames may be better
